OrbitCore/ScopeTimer.h

```diff
@@ -60,14 +60,19 @@ class Timer {
     UNREAL_OBJECT,
     ZONE,
     ALLOC,
-    FREE
+    FREE,
+    INTROSPECTION,
   };
 
   Type GetType() const { return m_Type; }
   void SetType(Type a_Type) { m_Type = a_Type; }
   bool IsType(Type a_Type) const { return m_Type == a_Type; }
   bool IsCoreActivity() const { return m_Type == CORE_ACTIVITY; }
 
+  char* EncodedString(){ return reinterpret_cast<char*>(&m_UserData); }
+  const char* EncodedString() const { return reinterpret_cast<const char*>(&m_UserData); }
+  void EncodeString(const char* msg) { strncpy(EncodedString(), msg, sizeof(m_UserData)-1); }
```

Comment:
If I read this correctly, then strings can only have 15 characters at most (+ terminating 0). I suggest adding a TODO to fix this with the StringManager that you suggested.